### PR TITLE
MTP-1555 legge til nyhetsbrev ikon

### PR DESCRIPTION
**Pull request by petterkoding**
Add Newsletter icon bell

**Designsystem**
- Added Newsletter button with icon  to _Stories, component > button > Secondary Large
- Added to visual style > icons

### CODE REVIEW

File: stories/components/Buttons.stories.svelte [hunk 1]
```diff
@@ -37,6 +37,8 @@
     <p class="description">Sekundær - stor</p>
     <button class="button button--secondary" {disabled}>{secondary}</button>
 
+    <button class="button button--secondary icon--newsletter-bell-before" {disabled}>Abonner på nyhetsbrev</button>
```
Comment (petterkoding) [inline stories/components/Buttons.stories.svelte:40]: @lha047 would you like to keep this Newsletter story here (Buttons) or move it to Links.

Comment (lha047) [reply stories/components/Buttons.stories.svelte:40]: You can put it in the Links story

Comment (petterkoding) [reply stories/components/Buttons.stories.svelte:40]: silly me used a `<button>` instead of `<a>` . Will fix :D

File: src/scss/global/_icon.scss [hunk 2]
```diff
@@ -21,6 +23,25 @@ $icon-gap: var(--spacer-xxx-small);
...
 
 .icon {
+
+  &--newsletter-bell-before{
+    @extend .icon-align;
+    &:before{
+      background-image: $icon-newsletter-blue;
+      padding-right: var(--icon-size-small);
+      content: "";
+      width: var(--icon-size-small);
```
Comment (amundlindstol) [inline src/scss/global/_icon.scss:33]:
```suggestion
      width: var(--icon-size-normal);
```

File: src/scss/global/_icon.scss [hunk 3]
```diff
@@ -21,6 +23,25 @@ $icon-gap: var(--spacer-xxx-small);
...
+      height: var(--icon-size-small);
```
Comment (amundlindstol) [inline src/scss/global/_icon.scss:34]:
```suggestion
      height: var(--icon-size-normal);
```

File: src/scss/global/_icon.scss [hunk 4]
```diff
@@ -21,6 +23,25 @@ $icon-gap: var(--spacer-xxx-small);
...
+      background-size: var(--icon-size-small) var(--icon-size-small);
```
Comment (amundlindstol) [inline src/scss/global/_icon.scss:35]:
```suggestion
      background-size: var(--icon-size-normal) var(--icon-size-normal);
```

**Review by lha047 — APPROVED** [review]
LGTM :D

File: stories/visualStyle/Icons.stories.svelte [hunk 5]
```diff
@@ -32,6 +32,10 @@
       <p class="icon--alert-filled-success-before">Medhold</p>
       <p class="icon--alert-filled-warning-before">Delvis medhold</p>
     </section>
+    <section>
+      <h2>Ikoner til Nyhetsbrev</h2>
```
[the remark this answers is not in the document]
Comment (amundlindstol) [reply stories/visualStyle/Icons.stories.svelte:36]: @petterkoding committing this so you don't forget

**Review by amundlindstol — APPROVED** [review]
good job :)

**Pull request merged**